src/features/all-drops/components/AllDrops.tsx

```diff
@@ -243,6 +244,11 @@ export default function AllDrops() {
     }, []);
   };
 
+  const createADropPopover = (menuIsOpen: boolean) => ({
+    header: 'Click here to start creating a drop!',
```

Comment:
```suggestion
    header: 'Click here to create a drop!',
```
would suggest to keep in short